lists/malware.h

```diff
@@ -27,6 +27,22 @@ const std::vector<FilterList> malware_lists = {
     "https://disconnect.me/",
     "",
     ""
+  }), FilterList({
+    "E5484F0C-F35C-11E8-8EB2-F2801F1B9FD1",
+    "https://gitcdn.xyz/repo/NanoMeow/MDLMirror/master/hosts.txt",
+    "Malware Domain List",
+    {},
+    "http://www.malwaredomainlist.com/",
```

Comment:
Their website used to show [list updates](https://www.malwaredomainlist.com/update.php) but the last one is from 2017-12-04. Their [forum](https://www.malwaredomainlist.com/forums/) looks abandoned too. The last post was 2018-05-09 and it was a report of a malware site by a user.

The [download link](https://gitcdn.xyz/repo/NanoMeow/MDLMirror/master/hosts.txt) included in this PR ultimately points to this fairly new [GitHub repo](https://github.com/NanoMeow/MDLMirror). Looking at that repo, the [first commit](https://github.com/NanoMeow/MDLMirror/commit/6750c41a82e70666240020f582eec87193fbbf70) which imported the list, was on 2019-01-04. The only [other commit which has changed the list since then](https://github.com/NanoMeow/MDLMirror/commit/f2bf59b7f6cfa20cc36aa6b41879fdb126ee9d29) was on 2019-02-20. It looks like it's still maintained, but it's not particularly active.